source/Nevermore/Advanced/QueryBuilder.cs

```diff
@@ -113,6 +113,20 @@ public IQueryBuilder<TRecord> CalculatedColumn(string expression, string columnA
             return this;
         }
 
+
+        public IQueryBuilder<TRecord> CountColumn(string columnAlias)
+        {
+            selectBuilder.AddColumnSelection(new AliasedColumn(new CalculatedColumn(new CustomExpression("COUNT (*)")), columnAlias));
```

Comment:
we could reuse the overload that we're decorating here, to minimise our reasons to change.
```suggestion
            selectBuilder.CalculatedColumn("COUNT (*)", columnAlias);
```